nbs/src/core/models.ipynb

```diff
@@ -74,6 +74,7 @@
     "import numpy as np\n",
```

Comment:
<p>Are we missing the docs at the end? Meaning </p><p> </p><p><code>show_doc(TBATS, title_level=3)</code></p><p><code>show_doc(ARCH.fit, name='ARCH.fit', title_level=3)</code></p><p> </p><p>etc.</p><p> </p><p>All other models have them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='465'/>